indexer.py

```diff
@@ -79,6 +81,7 @@ def __init__(self, blocks_per_call=None):
             self.index_apy: settings.INDEXER_STATS_BLOCKS_PER_CALL,
             # 268 blocks is roughly every hour on current Ethereum mainnet
             self.reset_apy_calc: 268,
+            self.index_strategy_balances: 1,
```

Comment:
Every block? :astonished:

Reply:
Used for testing locally, forgot to change back. Thanks!